reana_server/templates/emails/email_confirmation.txt

```diff
@@ -0,0 +1,9 @@
+Dear {{ user_full_name }},
+
+Your {{ reana_hostname }} account has been created successfully. Please confirm your email address by accessing this link <https://{{ reana_hostname}}/confirm/{{ confirm_token }}>.
```

Comment:
I think some users might copy the link with ">." symbols mixed with the token. One way to fix it would be to change it like this: "...by accessing this link: https... ". Also, maybe the link could be made clickable?

Reply:
We've got this same format in [other templates](https://github.com/reanahub/reana-server/blob/master/reana_server/templates/emails/token_granted.txt), that's why I kept it consistent. I guess eventually we can move to HTML templates and introduce proper links.

Reply:
The `<...>` format for links in plain text emails is a standard that most clients support out of the box. 

But I see it could create confusion for some users perhaps... An alternative would be to print URLs clearly on a new line separated from the main text (and without using brackets this time), such as:

```
Dear {{ user_full_name }}:

Your {{ reana_hostname }} account has been created successfully. 
Please confirm your email address by accessing the following link:

  https://{{ reana_hostname}}/confirm/{{ confirm_token }}

% include 'emails/assistance_links.txt' %}

Best regards,

REANA notification system
```

P.S. We can also pay attention to have the regular text wrapped into under 78 characters per line, as per RFC 2822.

P.S. ... and let's keep plain text emails over HTML emails please :wink: